packages/split-view/stories/split-view.stories.ts

```diff
@@ -19,14 +19,21 @@ export default {
     component: 'sp-split-view',
 };
 
-export const Horizontal = (): TemplateResult => {
+interface Properties {
+    primarySize?: number;
```

Comment:
If this was `string` could we then input `80%`, etc.? Is that helpful?

Reply:
Oh, yeah I wanted to change that, but then I forgot. Thanks